autopilot-daemon/pkg/cmd/main.go

```diff
@@ -10,12 +11,45 @@ import (
 
 	"github.com/IBM/autopilot/pkg/handlers"
 	"github.com/IBM/autopilot/pkg/utils"
-	"k8s.io/klog/v2"
-
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/tools/cache"
+	toolswatch "k8s.io/client-go/tools/watch"
+	"k8s.io/klog/v2"
 )
 
+func watchNodes() {
```

Comment:
Would you add a comment describing the functionality? Does it really watch node_s_ ... Maybe it should be named something like `watchForNodeChanges` b/c it's only looking at one node?